app/policies/participant_policy.rb

```diff
@@ -1,11 +1,26 @@
 # frozen_string_literal: true
 
-class ParticipantPolicy < ApplicationPolicy
+class ParticipantPolicy < UserPolicy
   def index?
-    admin_only
+    return true if admin_only
+    return @record.all? { |participant| user_can_access?(participant) } if user.induction_coordinator?
   end
```

Comment:
Ah, the beauty of Pundit. So we authorize `index` by passing collection of records as a record and then for each of this record we're loading its school (so we must rely on the policy internals being leaked outside of the policy) and we reject the entire view when a single record is wrong?

Again, this is not something to be addresses here (expecially given a massive amount of PR pending on this one), but  ideally we would not perform per-record authorisation here and make use of a Scope object inside the controller.

Reply:
On another note - could we have `return false unless user.induction_coordinator?` instead

Reply:
To be fair I have absolutely no idea if this is the best way to do this - the docs seem very unclear on what to do with indexes and I couldn't find any existing examples in our codebase

Reply:
For a list of admins we do
```
      def index
        @administrators = policy_scope(User).admins.order(:full_name)
      end
```
which could probably work in here too with some tweaking.

In any case, I think we should go with something simple that works here - and worry about improving it in a separate PR that doesn't block further work :)